src/main/kotlin/app/revanced/patches/facebook/ads/mainfeed/HideSponsoredStoriesPatch.kt

```diff
@@ -0,0 +1,42 @@
+package app.revanced.patches.facebook.ads.mainfeed
+
+import app.revanced.util.exception
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructionsWithLabels
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.facebook.ads.mainfeed.fingerprints.GetStoryVisibilityFingerprint
+import app.revanced.patches.facebook.ads.mainfeed.fingerprints.GraphQLStorySponsoredDataGetterFingerprint
+
+@Patch(
+    name = "Hide Sponsored stories",
+    description = "Hide Sponsored stories in main feed.",
+    compatiblePackages = [CompatiblePackage("com.facebook.katana")]
+)
+@Suppress("unused")
+object HideSponsoredStoriesPatch : BytecodePatch(setOf(GetStoryVisibilityFingerprint, GraphQLStorySponsoredDataGetterFingerprint)) {
+    override fun execute(context: BytecodeContext) {
+        GetStoryVisibilityFingerprint.result?.apply {
+
+            // Get the sponsored data model getter
```

Comment:
Comments end with punctuation
```suggestion
            // Get the sponsored data model getter.
```